services/parser-database/connector.py

```diff
@@ -61,6 +62,33 @@ def get_articles_that_match_keywords(keywords_list):
     return matching_articles
 
 
+def get_articles_by_tfidf_value(keywords_list):
+    """
+    Returns a value for every article based on a keyword
+    for a keyword list, value is based on
+    term frequency inverse document frequency (tfidf)
+    Args:
+        keywords_list (list): Keyword(s) to look for
+
+    Returns:
+        list: articles and value for such keyword(s)
+    """
+    matching_articles = {}
+    for keyword in keywords_list:
+        articles_that_match_keyword = {}
+        if keyword in keywords_in_memory:
+            for article in keywords_in_memory[keyword]:
+                # tfidf computation
+                term_density_in_article = article["frequency"]/article["wordCount"]
+                document_frequency = len(articles_in_memory)/len(keywords_in_memory[keyword])
+                inverse_doc_freq = np.log(document_frequency)
+                value = term_density_in_article * inverse_doc_freq
```

Comment:
**is dis advanced math?**
![isdis](https://user-images.githubusercontent.com/40501486/90733755-9cae6980-e292-11ea-8113-94d300529607.jpg)